server/src/main/java/io/spine/server/delivery/DeliveryMonitor.java

```diff
@@ -95,6 +95,34 @@ public void onDeliveryStarted(ShardIndex index) {
         // do nothing.
     }
 
+    /**
+     * Called if an {@code Exception} occurred when the {@code Delivery} attempted
+     * to pick up a shard.
+     *
+     * <p>Pay attention, this method is not called if the shard could not be picked up because
```

Comment:
Let's improve:
```
<p>Please note, this callback is <em>not</em> invoked in case the shard cannot be picked
if it's already picked by another worker. It is so, because such a use case is a part of normal flow,
and thus does not provoke a {@code RuntimeException}.
```